src/chainlit/frontend/src/components/organisms/slider.tsx

```diff
@@ -1,10 +1,35 @@
 import { grey } from 'palette';
 
-import { Box, Stack } from '@mui/material';
-import Slider, { SliderProps } from '@mui/material/Slider';
+import Slider, { SliderProps as MSliderProps } from '@mui/material/Slider';
 import { styled } from '@mui/material/styles';
 
-import InputLabel from 'components/molecules/inputLabel';
+import { IInput } from 'types/Input';
+
+import InputStateHandler from './inputs/inputStateHandler';
+
+export type SliderProps = IInput & MSliderProps;
+
+const _Slider = ({
+  description,
+  hasError,
+  id,
```

Comment:
I noticed that some current Slider implementations now have errors since id is required